src/lib/ui/SupportedDevices/SupportedDeviceRow.tsx

```diff
@@ -0,0 +1,21 @@
+export default () => {
+  return (
+    <tr>
+      <td>
+        <img
+          style={{ width: 40 }}
+          src='https://www.seam.co/_next/image?url=%2Fimg%2Fdevice-db%2Faugust%2Fsmartlocks%2Faugust_wifi-smart-lock_silver_front.png&w=96&q=75'
+        />
+      </td>
+      <td>Smart Lock</td>
+      <td>Wi-Fi Smart Lock</td>
+      <td>AUG-SL05-M01-S01</td>
+      <td>Wi-Fi</td>
+      <td>
+        <div className="seam-status-pill status-green">
```

Comment:
This isn't in the official design, therefore not an atomic component on its own. This was included in the supported devices table on the marketing site, so I just added a tiny bit of UI and styling to add some color to it. Can easily be removed/extracted out if we like it.

<img width="126" alt="Arc 2023-05-22 at 15 16 47@2x" src="https://github.com/seamapi/react/assets/87919558/78024203-b736-4d09-9d40-4bb9b0f52d23">

Reply:
sgtm